frontend/src/components/maps/leaflet/LayerPopup/hooks/useLayerQuery.tsx

```diff
@@ -78,10 +80,40 @@ export const handleParcelDataLayerResponse = (
     .then((resp: FeatureCollection<Geometry, GeoJsonProperties>) => {
       saveParcelDataLayerResponse(resp, dispatch, latLng);
     })
-    .catch((axiosError: any) => {
+    .catch((axiosError: AxiosError) => {
       dispatch(error(parcelLayerDataSlice.reducer.name, axiosError?.response?.status, axiosError));
     });
 };
+let loadingToastId: React.ReactText | undefined = undefined;
+let errorToastId: React.ReactText | undefined = undefined;
+const MAX_RETRIES = 2;
```

Comment:
wonder if you can pull this into the other global constants

Reply:
its not really global, these retries are only for this one hook.